Header header

EngineState basic_state

uint16 rate # Speed
uint16 fuel_pressure # fuel pressure
uint16 fuel_temperature # Low Fuel
uint8 fuel_level_main_tank # main tank fuel level
uint8 fuel_level_aux_tank # additional level of fuel tank
uint16 fuel_level_auxiliary_tank # oil pressure
uint16 oil_temperature_in # oil temperature at the inlet
uint16 oil_temperature_out # oil temperature at the outlet
uint16 air_in_temperature # pumping of air temperature
uint16 cylinder1_temperature1 # 1 Low pneumatic cylinder
uint16 cylinder2_temperature2 # 2 Low pneumatic cylinder
uint16 cylinder3_temperature3 # Low pneumatic cylinder 3
uint16 cylinder4_temperature4 # Low air tsilindra4
uint16 cylinder5_temperature5 # 5 pneumatic cylinder temperature
uint16 cylinder6_temperature6 # Low air tsilindra6
uint16 supercharged_temperature # supercharged temperature
uint8 supercharged_pressure # 8 bits supercharged pressure
uint32 reserv # 
uint8 errors_cnt # of accidents
uint8 errors_flag1 # error code 
uint8 errors_flag2 # error code 